web/middlewares/instance.go

```diff
@@ -63,12 +64,11 @@ func CheckInstanceBlocked(next echo.HandlerFunc) echo.HandlerFunc {
 
 			// Fallback by trying to determine the blocking reason
 			reason := i.BlockingReason
-			returnCode := http.StatusPaymentRequired
 
 			if reason == "" {
-				reason = http.StatusText(http.StatusPaymentRequired)
+				reason = i.Translate(instance.BlockedUnknown.Message)
 			} else if reason == instance.BlockedPaymentFailed.Code {
-				reason = instance.BlockedPaymentFailed.Message
+				reason = i.Translate(instance.BlockedPaymentFailed.Message)
```

Comment:
Shouldn't we have `returnCode = http.StatusPaymentRequired` here?

Reply:
Thanks, fixed ;)